pkgs/tools/security/cryptomator/default.nix

```
@@ -102,7 +102,7 @@ in stdenv.mkDerivation rec {
     autoPatchelfHook
     maven
     makeWrapper
-    (wrapGAppsHook.override { makeBinaryWrapper = makeWrapper; })
+    (wrapGAppsHook.override { inherit makeWrapper; })
```

```suggestion
    wrapGAppsHookp
```
Just replace makeWrapper with makeShellWrapper.

---

Indeed, that seems better.